<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>堇了</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>状态</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radBtn_type_LM">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>117</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>蓝莓</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radBtn_type_MN">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>117</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>猛男</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radBtn_status_1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>117</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>我觉得可以</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radBtn_status_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>117</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>我觉得还行</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radBtn_status_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>120</y>
     <width>117</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>我觉得不行</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>阶段</string>
   </property>
   <widget class="QRadioButton" name="radBtn_gpBox_1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>萌新</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radBtn_gpBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>大佬</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radBtn_gpBox_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>腊肉</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
